Sharpmake.Generators/VisualStudio/Androidproj.cs

```diff
@@ -346,13 +355,20 @@ private Options.ExplicitOptions GenerateOptions(AndroidPackageProject project, s
                     if (appLibConf.Output != Project.Configuration.OutputType.Dll)
                         throw new Error("Cannot use configuration \"{0}\" as app lib for package configuration \"{1}\". Output type must be set to dynamic library.", appLibConf, conf);
 
-                    options["OutputFile"] = appLibConf.TargetFileFullName;
+                    if (!appLibConf.TargetFileFullName.StartsWith("lib"))
+                        throw new Error("Android libraries must start with the prefix 'lib' so they can be referenced without it. Configuration \"{0}\" contains filename \"{1}\".", appLibConf, appLibConf.TargetFileFullName);
```

Comment:
Not sure this is very clear. Rules have changed about libraries with commit 93c98ee7ab0416af7e1bafc43e11689bb8facc57. Please clarify the error message.

Reply:
This was unnecessary, I was add "lib" prefix in my projects when it really needed to be added to the android platform.